cluster/kubemark/config-default.sh

```diff
@@ -33,9 +33,9 @@ MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-container-vm-v20150806}
 MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-google-containers}
 
 NETWORK=${KUBE_GCE_NETWORK:-default}
-INSTANCE_PREFIX="hollow-kubernetes"
-MASTER_NAME="hollow-cluster-master"
-MASTER_TAG="hollow-cluster-master"
+INSTANCE_PREFIX="${INSTANCE_PREFIX:-"default"}"
```

Comment:
E2E_CLUSTER_NAME is set only by Jenkins - I'd rather have some sane default without using environmental variables, if possible.